report-api/src/main/resources/template/nvi-institution-status.sparql

```diff
@@ -0,0 +1,33 @@
+PREFIX : <https://nva.sikt.no/ontology/publication#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+SELECT DISTINCT
+  ?STATUS_KONTROLLERT
+  ?PUBLIKASJONSFORM
+  ?PUBLISERINGSKANALTYPE
+  ?ISSN
+  ?KVALITETSNIVAKODE
+  ?PERSONLOPENR
+  ?INSTITUSJONSNR
+  ?AVDNR
+  ?UNDAVDNR
+  ?GRUPPENR
+  ?FORFATTERE_TOTALT
+  ?FORFATTERE_INT
+  ?VEKTET
+  ?ETTERNAVN
+  ?ETTERNAVN
+  ?FORNAVN
+  ?PUBLISERINGSKANALNAVN
+  ?SIDE_FRA
+  ?SIDE_TIL
+  ?SIDEANTALL
+  ?VA_TITTEL
+  ?SPRAK
+  ?RAPPORTERT_STATUS
+  ?VEKTINGSTALL
+  ?FAKTORTALL_SAMARBEID
+  ?FORFATTERDEL
+  ?FORFATTERVEKT
```

Comment:
Ugh…ok for now, but I'd prefer an understandable description of the values followed by a mapping to the required shorthand in the OOXML

Reply:
Agreed, will add an understandable description when i add the query